pipeline/outputs/opensearch.md

```diff
@@ -157,3 +158,21 @@ Without this you will see errors like:
 ```text
 {"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"Action/metadata line [1] contains an unknown parameter [_type]"}],"type":"illegal_argument_exception","reason":"Action/metadata line [1] contains an unknown parameter [_type]"},"status":400}
 ```
+
+### Amazon OpenSearch Serverless <a id="#aws-opensearch-serverless"></a>
+Amazon OpenSearch Serverless an offering that eliminates your need to manage OpenSearch clusters. All existing Fluent Bit OpenSearch output plugin options work with OpenSearch Serverless. The only difference with serverless from a Fluent Bit POV is that you must specify the service name as `aoss` (Amazon OpenSearch Serverless) when you enable `AWS_Auth`:
```

Comment:
For Fluent Bit, the only difference is that you must specify the service name...

Reply:
Updated